### ISSUE TITLE
Привести документацию оформления описания в корректный вид

### ISSUE THREAD

**Issue by Kyrales** [zerobig/swagger-1c#19]
См. читабельный вариант оформления

![image](https://github.com/user-attachments/assets/9cdd8caa-5899-40a4-8af6-fcb52bae8eba)

Дополнительно шаблоны быстрого доступа добавить к примерам:
1. Добавить свойство объекта (схема)
.Свойство("<?>")
	.Описание("КраткоеОписание. В 1С: НазваниеОбъекта.ИмяРеквизита")
	.ТипЗначения("<?"Выберете тип свойства", ВыборВарианта, "string", "string", "integer", "integer", "boolean", "boolean", "number", "number">")
	.Пример("123") 

2. Добавить свойство объекта (схема)
			.Свойство("<?>")
				.Описание("")
				.ТипЗначения("<?"Выберете тип свойства", ВыборВарианта, "object", "object", "array", "array">")
				.Схема("ОбъектСсылкаПример")
		.Объект("ОбъектСсылкаПример")  // дочерний объект свойства ИмяСвойства
			.Свойство("Идентификатор")
				.Описание("Идентификатор договора из 1С. В 1С: НазваниеОбъекта.ллл_КодРС")
				.ТипЗначения("string")
				.Пример("8f80d9d3-730a-4b8f-82a5-4ff52960691e")

**Comment by Kyrales** [comment]
Указанные шаблоны ускоряют работу со свойствами

**Comment by Kyrales** [comment]
Скоро еще дополню полноценным шаблоном st

**Comment by zerobig** [comment]
Как-то вот так?
[swagger-1c.zip](https://github.com/user-attachments/files/18019918/swagger-1c.zip)

**Comment by Kyrales** [comment]
Да, такой набор разных свойств можно включить в базовую документацию
